src/Mod/TechDraw/Gui/TaskDimension.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>330</width>
-    <height>417</height>
+    <height>706</height>
```

Comment:
This looks strange. I cannot compile this right now to have a look but 700 px is definitely too large for e.g. my laptop screen I use to view and modify drawings

Reply:
The widget gets a scroll bar.